shared/src/userData.ts

```diff
@@ -106,6 +106,7 @@ export interface Preferences {
   isCalendarFullView: boolean;
   isHideableRoadmapOpen: boolean;
   phaseNewlyChanged: boolean;
+  isNonProfitFromFunding?: boolean;
```

Comment:
sand - we've added undefined values like `registeredISO: undefined`, shouldn't we add the two newly created valued?